gnucash/gnome/gnc-plugin-page-sx-list.c

```diff
@@ -606,7 +606,8 @@ gnc_plugin_page_sx_list_cmd_new(GtkAction *action, GncPluginPageSxList *page)
         schedule = g_list_append(schedule, r);
         gnc_sx_set_schedule(new_sx, schedule);
     }
-    gnc_ui_scheduled_xaction_editor_dialog_create(new_sx, new_sx_flag);
+    gnc_ui_scheduled_xaction_editor_dialog_create(gnc_ui_get_main_window (NULL),
```

Comment:
Plugin pages are always part of a main window somehow. So you can get their main_window via
```
GTK_WINDOW (gnc_plugin_page_get_window (GNC_PLUGIN_PAGE (plugin_page)));
```

 There should be no need to call gnc_ui_get_main_window here.